runtime/init.go

```diff
@@ -106,7 +102,18 @@ func (d *Default) initRunner(cluster *v1.Cluster) error {
 
 	return nil
 }
-
+func (d *Default) CopyMetadata() error {
```

Comment:
Why we need copy Metadata?

Reply:
at init,applyNodes,applyMasters process need Metadata,  but Metadata in rootfs   and read metadata not mount /tmp/<cluster-name> yet , read metada at runtime init func so far.

Reply:
Using /var/lib/sealer/../Metadata

Reply:
yes, i using it .but in runtime init this dir not exists  yet

Reply:
Newruntime after Mount image, so that you needn't copy metadata